app/src/main/java/com/eventyay/organizer/core/event/create/UpdateEventFragment.java

```diff
@@ -317,7 +324,7 @@ private void setupPlacesAutocomplete() {
 
         binding.form.selectLocationButton.setOnClickListener(view -> {
 
-            if (mapboxAccessToken.equals("YOUR_ACCESS_TOKEN")) {
+            if ("YOUR_ACCESS_TOKEN".equals(mapboxAccessToken)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/app/fossasia/open-event-orga-app/pullRequest?prid=4593769)